main.lua

```diff
@@ -89,6 +89,8 @@ local function start(...)
     .alias({['A'] = 'auto-create-entity'})
     .describe("N", "do not prompt for entity creation")
     .alias({['N'] = 'no-entity'})
+    .describe("ca", "location of the CA you want to provide")
```

Comment:
Thinking about this more, it's good to avoid "you" in user facing docs. Perhaps re-word this to "path for a custom CA certificate file in PEM format"